onnxruntime/python/tools/kernel_explorer/README.md

```diff
@@ -1,12 +1,37 @@
-# Kernel Explorer 
+# Kernel Explorer
+
 Kernel Explorer hooks up GPU kernel code with a Python frontend to help develop, test, profile, and auto-tune GPU kernels.
 
-## Example Usage
-```
-mkdir build
-cd build
-cmake -D onnxruntime_ROCM_HOME=/opt/rocm ..
-cmake --build .
-pytest ../kernels --verbose
-python ../kernels/vector_add_test.py
+The initial scope is for BERT-like models with ROCM EP.
+
+## Build
```

Comment:
I gave this a try, got the error below:

CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.

Reply:
You just install ninja, maybe via apt.